src/nomad_material_processing/general.py

```diff
@@ -449,36 +449,71 @@ class Miscut(ArchiveSection):
     The miscut in a crystalline substrate refers to
     the intentional deviation from a specific crystallographic orientation,
     commonly expressed as the angular displacement of a crystal plane.
+
+    It can be either directed along one of the crystallographic axes or in a specific
+    direction in the crystal defined by a main direction and the perpendicular one.
```

Comment:
But do we actually have a way of specifying this now? As far as I can see we now only have one direction and an angle. Tbh, I'm a bit confused now on what Oliver was talking about, maybe we need to have a call to discuss this.